utils/copy-env-vars.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
Can we add `utils/**` to all `.dockerignore` please.

Reply:
Please use the same bash script pattern as `deploy.sh`.

Reply:
this is the same :)

Reply:
Also -- none of the files should be getting copied by our docker file currently